mod/display.php

```diff
@@ -96,7 +96,7 @@ function display_init(App $a)
 	}
 
 	if ($item["id"] != $item["parent"]) {
-		$item = Item::selectFirstForUser(local_user(), $fields, ['id' => $item["parent"]]);
+		$item = Item::selectFirstForUser($item_user, $fields, ['id' => $item["parent"]]);
```

Comment:
I think the ```local_user()``` should stay here, since the whole blocking needs to be done for the given viewer.

Reply:
Otherwise the `$item_user` isn't used. It is one of the prospective change that makes more sense than removing the variable, but maybe it was just a development artifact.

Reply:
Yeah, possibly we can replace ```local_user()``` with ```$item_user``` here.